build.gradle

```diff
@@ -33,10 +33,10 @@ ext.libProperties.load(file("${rootDir}/lib.properties").newReader())
 
 ext {
     appcompatVersion = '1.1.0'
-    retrofitVersion = '2.6.1'
-    coroutinesVersion = '1.3.2'
-    junitVersion = '4.12'
-    mockitoVersion = '1.10.19'
+    retrofitVersion = '2.8.1'
+    coroutinesVersion = '1.3.5'
```

Comment:
there is at least 1.3.7

Reply:
You are right, fixed in 185a5844bbb8b498a2c89bcb1a881d83031892e8. AS knew only about 1.3.5 🤔